pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -102,16 +166,62 @@ pub struct ChargeFrqTransactionPayment<T: Config>(#[codec(compact)] BalanceOf<T>
 
 impl<T: Config> ChargeFrqTransactionPayment<T>
 where
-	BalanceOf<T>: Send + Sync + FixedPointOperand,
+	BalanceOf<T>: Send + Sync + FixedPointOperand + IsType<CapacityBalanceOf<T>> + From<u64>,
```

Comment:
Both CapacityBalanceOf and BalanceOf are associate types that come from the Currency trait. Although they look the same because they are defined in different places to rust, they are two different types. If I understand correctly, this trait allows us to say that these are technically the same thing.